src/org/thoughtcrime/securesms/WelcomeActivity.java

```diff
@@ -60,6 +61,16 @@ public void onCreate(Bundle bundle) {
         DcEventCenter eventCenter = DcHelper.getEventCenter(this);
         eventCenter.addObserver(DcContext.DC_EVENT_CONFIGURE_PROGRESS, this);
         eventCenter.addObserver(DcContext.DC_EVENT_IMEX_PROGRESS, this);
+
+        if (!DcHelper.hasAnyConfiguredContext(this)) {
+          Intent intent = new Intent();
+          // Since android API 26 only explicit broadcasts are allowed for IPC with a few exceptions.
+          // As a result we have to send for each companion app we want to support an intent with a
+          // specified package
+          intent.setPackage("org.cyberta");
```

Comment:
I was also thinking about renaming the package structure in AndroidYggmail. I wonder if it's appId should rather be chat.delta.AndroidYggmail.

Reply:
I think I'd slightly prefer `chat.delta.AndroidYggmail`.